eng/testing/linker/project.csproj.template

```diff
@@ -1,4 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project>
+
+  <PropertyGroup>
+    <!-- Needed for PublishTrimmed -->
+    <ToolsILLinkDir>{ToolsILLinkDir}</ToolsILLinkDir>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
```

Comment:
Do we really need to "break out" of using `Sdk="Microsoft.NET.Sdk"` here? Can we just move the `Directory.Build.props` change into this file (so all these "locate local tools" is contained in one place)?